packages/opentelemetry-remote-sampler-aws-xray/src/remote-sampler.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Sampler, SamplingDecision, SamplingResult } from '@opentelemetry/sdk-trace-base';
+import { SamplingRule } from './remote-sampler.types';
+import axios from 'axios';
+
+const DEFAULT_INTERVAL = 5 * 60 * 1000;// 5 minutes on sampling rules fetch (default polling interval)
+
+
+// IN PROGRESS - SKELETON CLASS 
+export class AWSXRayRemoteSampler implements Sampler {
+    private _pollingInterval: number;
+    private _endpoint: string;
+    private _samplingRulesEndpoint: string;
+
+    constructor(endpoint: string, pollingInterval: number = DEFAULT_INTERVAL) {
+
+        if (pollingInterval <= 0 || !Number.isInteger(pollingInterval)) {
+            throw new TypeError('pollingInterval must be a positive integer');
+        }
```

Comment:
I wonder if it is required to verify it's an integer. The `setInterval` function [receives a number that is not enforced to be an integer](https://nodejs.org/api/timers.html#setintervalcallback-delay-args):

> Non-integer delays are truncated to an integer

I think it will work as expected even when interval is not an integer, and we don't gain much from this complexity.